docs/customizations/operation-links.md

```
@@ -0,0 +1,67 @@
+| `method` | `string` | `''` | The HTTP method (GET, POST, etc.) |
+| `title` | `string` | `''` | The text to display for the link |
+
+### Usage
+
+To add an operation link in your Vue template, use the component as shown below:
+
+```vue
+<OAOperationLink href="/example/operations/getAllArtists" method="get" title="Get all artists" />
+```
+
+This will render a clickable link with a method badge: <OAOperationLink href="/example/operations/getAllArtists" method="get" title="Get all artists" />
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: OAOperationLink

(MD033, no-inline-html)